js/utils/serverConnect.js

```diff
@@ -44,7 +44,8 @@ function log(msg) {
 
   const logMsg = `[SERVER-CONNECT] ${msg}${EOL}`;
   debugLog += logMsg;
-  ipcRenderer.send('server-connect-log', logMsg);
+  // "if (ipcRenderer)" needed to prevent the `npm test` from bombing.
```

Comment:
I really, really, really dislike having to modify a source file to accommodate a test, but ipcRenderer is undefined during testing and I can't seem to be able to stub it (using sinon or otherwise). I'll probably circle back to this, but it works for now.